README.md

```diff
@@ -22,6 +22,21 @@ paru -S waycorner
 cargo install waycorner --locked
 ```
 
+### Nix/NixOS
+
+[`waycorner` package]: https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/misc/waycorner/default.nix
+[search.nixos.org]: https://search.nixos.org/packages?channel=unstable&show=waycorner&from=0&size=50&sort=relevance&type=packages&query=waycorner
+[@NotAShelf]: https://github.com/notashelf
+[out-of-date package report]: https://github.com/NixOS/nixpkgs/issues/new?assignees=&labels=9.needs%3A+package+%28update%29&projects=&template=out_of_date_package_report.md&title=Update+request%3A+PACKAGENAME+OLDVERSION+%E2%86%92+NEWVERSION
+
+There is a [waycorner package] package available via Nixpkgs. Refer to [search.nixos.org] page for Waycorner
+for the appropriate installation methods.
```

Comment:
```suggestion
The [`waycorner`] package is available via Nixpkgs. [See installation instructions.]
```

The link above is not the same so this is not rendering. Also fixed some phrasing weirdness.